projects/ngrx.io/content/marketing/docs.md

```diff
@@ -2,14 +2,55 @@
 
 NgRx is a framework for building reactive applications in Angular. NgRx provides state management, isolation of side effects, entity collection management, router bindings, code generation, and developer tools that enhance developers experience when building many different types of applications.
 
-## Core Principles
-
-- State is a single, immutable data structure.
-- Components delegate responsibilities to side effects, which are handled in isolation.
-- Type-safety is promoted throughout the architecture with reliance on TypeScript's compiler for program correctness.
-- Actions and state are serializable to ensure state is predictably stored, rehydrated, and replayed.
-- Promotes the use of functional programming when building reactive applications.
-- Provide straightforward testing strategies for validation of functionality.
+## Why NgRx?
+
+NgRx is built for creating maintainable explicit applications, by storing single state and the use of actions in order to express state changes.
+
+
+### Serializability
+
+By normalizing state changes and pass them through observables NgRx provides serializability and ensure state is predictably stored. This enables to save states to the `localStorage` or any other external storage.
+In addition, it also allows to inspect, download, upload, and dispatch actions, all from the [Store Devtools](guide/store-devtools).
+
+
+### Type Safety
+Type-safety is promoted throughout the architecture with reliance on TypeScript's compiler for program correctness.
+
+
+### Encapsulation
+
+Using Ngrx [Effects](guide/effects) and [Store](guide/store), any interaction with external resources side effects, like network requests, web socket and any business logic can be isolated from the UI. This isolation allows for more pure and simple components, and keep the single responsibility principle.
+
+### Testable
+
+Since NgRx is mostly based on pure functions like `selectors` and `reducers`, plus the ability to isolate side effects from the UI, testing becomes very straightforward.
+NgRx also provides tests setup like `provideMockStore` and `provideMockActions` for better test experience.
+
+### Performance
+
+NgRx is built on a single immutable data state, making change detection turn into a very easy task, which means `OnPush` strategy becomes available.
```

Comment:
What about referring to the Angular `OnPush` docs here?

Reply:
I can remove it, but I thought it could be a good reference for newcomers to explore

Reply:
Done :)